tests/install_upgrade_operators/csv/test_subscription_channels.py

```diff
@@ -11,5 +11,28 @@ def test_channels_in_manifest(kubevirt_package_manifest_channels):
 
 
 @pytest.mark.polarion("CNV-11944")
-def test_cnv_subscription_channel(cnv_subscription_scope_session, kubevirt_package_manifest_current_channel):
-    assert cnv_subscription_scope_session.instance.spec.channel == kubevirt_package_manifest_current_channel.name
+def test_cnv_subscription_channel(
+    cnv_subscription_scope_session, kubevirt_package_manifest_channels, cnv_current_version
+):
+    subscription_channel = cnv_subscription_scope_session.instance.spec.channel
+
+    assert subscription_channel, (
+        f"CNV subscription '{cnv_subscription_scope_session.name}' does not have a channel specified. "
+        f"Subscription spec.channel is: {subscription_channel!r}"
+    )
+
+    available_channels_for_version = [
```

Comment:
each verification should be a different test:
1. a test verifies one thing
2. when something fails, it is clear what functionality failed
3. when a test fails, the other verifications are still tested

Reply:
I see. I will simplify the asserts then, no need to add new tests IMO.